### DISCUSSION TITLE
vue/nodejs/%E4%BB%8B%E7%BB%8D%E4%B8%8E%E5%AE%89%E8%A3%85

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#753] [Announcements]
# vue/nodejs/%E4%BB%8B%E7%BB%8D%E4%B8%8E%E5%AE%89%E8%A3%85

一、node下载安装 官网下载 官网 (https://nodejs.org/en/download/) 根据自己电脑系统选择合适的版本进行下载，我这里选择window 64 位 点击安装 下载完点击安装 !点击安装 (/assets/vue/node/46659db54016445d94956b6350fd663c.png) !自定义路径 (/ass...

https://codelearning-9gtr246hb9b78416-1316243198.tcloudbaseapp.com/vue/nodejs/%E4%BB%8B%E7%BB%8D%E4%B8%8E%E5%AE%89%E8%A3%85.html

**Comment by cxslh** [comment]
奥里给
